helm/aws-load-balancer-controller/templates/deployment.yaml

```diff
@@ -153,8 +153,6 @@ spec:
           value: "{{ $value }}"
         {{- end }}
         {{- end }}
-        command:
-        - /controller
```

Comment:
Is there a standard entrypoint for ko built container images?

Reply:
Yes. See the last sentence of https://ko.build/get-started

Reply:
Thanks. The entrypoint change will introduce backwards incompatibility with the existing manifests, we will release note it.